DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -148,6 +59,67 @@ function WatchActors()
     foreach AllActors(class'#var(prefix)HKHangingPig',pig){
         AddWatchedActor(pig,"BeatTheMeat");
     }
+    foreach AllActors(class'#var(prefix)BarrelVirus',virus){
+        AddWatchedActor(virus,"WhyContainIt");
+    }
+    foreach AllActors(class'#var(prefix)Mailbox',mail){
+        AddWatchedActor(mail,"MailModels");
+    }
+    foreach AllActors(class'#var(prefix)CigaretteMachine',cigVending){
+        AddWatchedActor(cigVending,"SmokingKills");
+    }
+
+}
+
+function AddPhoneTriggers(bool isRevision)
+{
+    local #var(prefix)Phone p;
+    local BingoTrigger bt;
+    local int i;
+
+    //Spawn invisible phones for the payphones
+    switch(dxr.localURL) {
```

Comment:
you wanna know a really dumb trick you could do here? no idea if this is terrible practice or not

```
switch(dxr.localURL @ isRevision) {
case "02_NYC_STREET false":
```